gui/wxpython/gui_core/mapdisp.py

```diff
@@ -349,6 +402,47 @@ def StatusbarEnableLongHelp(self, enable=True):
         for toolbar in six.itervalues(self.toolbars):
             toolbar.EnableLongHelp(enable)
 
+    def ShowAllToolbars(self, show=True):
+        if not show:  # hide
+            action = self.RemoveToolbar
+        else:
+            action = self.AddToolbar
+        for toolbar in self.GetToolbarNames():
+            action(toolbar)
+
+    def AreAllToolbarsShown(self):
+        return self.GetMapToolbar().IsShown()
+
+    def GetToolbarNames(self):
+        """Return toolbar names"""
+        return self.toolbars.keys()
```

Comment:
To avoid potential problems like in #1763:
```suggestion
        return list(self.toolbars.keys())
```